packages/client/hmi-client/src/views/DocumentView.vue

```diff
@@ -0,0 +1,121 @@
+<script setup lang="ts">
+import Document from '@/components/articles/Document.vue';
+import TabContainer from '@/components/tabs/TabContainer.vue';
+import { ref, watch, computed } from 'vue';
+import { Tab } from '@/types/common';
+import useResourcesStore from '@/stores/resources';
+import { Project } from '@/types/Project';
+import { RouteName } from '@/router/routes';
+import { useTabStore } from '@/stores/tabs';
+import { isEmpty } from 'lodash';
+import ResourcesList from '@/components/resources/resources-list.vue';
+
+const props = defineProps<{
+	assetId?: string;
+	project: Project;
+}>();
+
+const emit = defineEmits(['show-data-explorer']);
+
+const resourcesStore = useResourcesStore();
+const tabStore = useTabStore();
+
+const newDocumentId = computed(() => props.assetId);
+const openTabs = ref<Tab[]>([]);
+const activeTabIndex = ref(0);
+const documentsInCurrentProject = resourcesStore.activeProjectAssets?.publications; // fixme
+const activeProject = resourcesStore.activeProject;
+const tabContext = `document${activeProject?.id}`;
```

Comment:
Should we add a `:` between `document` and the project Id.

Reply:
Imo the tab context doesn't really need to be nicely formatted or even human readable.

Reply:
We have a lot of convention-related matters we can talk about, I will make a note, but let's table this for now.